### PR TITLE
info about development environment dependencies

### PR DESCRIPTION
**Pull request by pdelewski**
More info regarding proper development environment requirements (https://github.com/open-telemetry/opentelemetry.io/issues/2184)

### CODE REVIEW

File: content/en/docs/instrumentation/php/automatic.md [hunk 1]
```diff
@@ -49,6 +49,13 @@ function ends it.
 
     * **pickle** can be used to install extensions that are available via http://pecl.php.net, however that's not the case
     for opentelemetry-php-instrumentation yet, so the only way for it is to install directly from source code.
+
+        **Installing from source requires proper development environment and few dependendencies:**
+
+        - `C compiler`
+        - `autoconf`
+        - `make`
```
Comment (svrnm) [inline content/en/docs/instrumentation/php/automatic.md:57]: would it be possible to provide some specific examples? For C++ we have tabs that give examples for some linux distros and macos:

https://opentelemetry.io/docs/instrumentation/cpp/getting-started/

Comment (pdelewski) [reply content/en/docs/instrumentation/php/automatic.md:57]: ok, thx. Will try to provide something similar.

**Comment by chalin** [comment]
Preview: https://deploy-preview-2192--opentelemetry.netlify.app/docs/instrumentation/php/automatic/#setup

**Review by chalin — CHANGES_REQUESTED** [review]
You'll need to fix the indentation of the text that you added, otherwise it "breaks" the layout as you can see here:

> image

Also, it'll probably be best to wait for #2192 to land first and then resubmit this change (possibly after running Prettier over the file).

**Comment by svrnm** [comment]
Preview:

https://deploy-preview-2193--opentelemetry.netlify.app/docs/instrumentation/php/automatic/

File: content/en/docs/instrumentation/php/automatic.md (hunk 1 above)
Comment (svrnm) [reply content/en/docs/instrumentation/php/automatic.md]: TY

File: content/en/docs/instrumentation/php/automatic.md [hunk 2]
```diff
@@ -57,6 +57,14 @@ function ends it.
...
       this case):
 
+      **Installing from source requires proper development environment and few
+      dependendencies:**
+
+      {{< ot-tabs "Linux (apt)" "MacOS (homebrew)">}}
+      {{< ot-tab lang="shell">}} $ sudo apt-get install gcc make autoconf {{< /ot-tab >}}
+      {{< ot-tab lang="shell">}} $ brew install gcc make autoconf {{< /ot-tab >}}
+      {{< /ot-tabs >}}
+
```
Comment (svrnm) [inline content/en/docs/instrumentation/php/automatic.md:67]:
```suggestion
With your environment setup you can install the extension:
```

The two code blocks look odd without being separated by a sentence, so here's a suggestion what to put inbetween

**Review by svrnm — APPROVED** [review]
lgtm

**Comment by cartermp** [comment]
@chalin can you give this another look? Your feedback was addressed and this should be good to go

File: content/en/docs/instrumentation/php/automatic.md [hunk 3]
```diff
@@ -57,6 +57,14 @@ function ends it.
       how to do that using a specific version of the extension (1.0.0beta2 in
       this case):
 
+      **Installing from source requires proper development environment and few
+      dependendencies:**
```
Comment (chalin) [inline content/en/docs/instrumentation/php/automatic.md:61]:
```suggestion
      Installing from source requires proper development environment and few
      dependencies:
```

File: content/en/docs/instrumentation/php/automatic.md [hunk 4]
```diff
@@ -57,6 +57,14 @@ function ends it.
...
+
+      {{< ot-tabs "Linux (apt)" "MacOS (homebrew)">}}
```
Comment (chalin) [inline content/en/docs/instrumentation/php/automatic.md:63]:
```suggestion
      {{< ot-tabs "Linux (apt)" "macOS (homebrew)">}}
```

File: content/en/docs/instrumentation/php/automatic.md [hunk 5]
```diff
@@ -57,6 +57,14 @@ function ends it.
...
+      {{< ot-tab lang="shell">}} $ sudo apt-get install gcc make autoconf {{< /ot-tab >}}
+      {{< ot-tab lang="shell">}} $ brew install gcc make autoconf {{< /ot-tab >}}
```
Comment (chalin) [inline content/en/docs/instrumentation/php/automatic.md:65]:
```suggestion
      {{< ot-tab lang="shell">}} sudo apt-get install gcc make autoconf {{< /ot-tab >}}
      {{< ot-tab lang="shell">}} brew install gcc make autoconf {{< /ot-tab >}}
```

File: content/en/docs/instrumentation/php/automatic.md [hunk 6]
```diff
@@ -57,6 +57,14 @@ function ends it.
...
+      {{< /ot-tabs >}}
+With your environment setup you can install the extension:
```
Comment (chalin) [inline content/en/docs/instrumentation/php/automatic.md:67]:
```suggestion
      {{< /ot-tabs >}}
      
      With your environment setup you can install the extension:

```

**Review by chalin — COMMENTED** [review]
See inline comments.

This should probably be using the Docsy tabbed panes rather than the OTel-custom shortcode.

File: content/en/docs/instrumentation/php/automatic.md (hunk 4 above)
Comment (pdelewski) [reply content/en/docs/instrumentation/php/automatic.md:63]: @chalin ok. I made changes based on https://github.com/open-telemetry/opentelemetry.io/blob/main/content/en/docs/instrumentation/cpp/getting-started.md. Seems that it should be updated as well to be consistent.

File: content/en/docs/instrumentation/php/automatic.md (hunk 5 above)
Comment (pdelewski) [reply content/en/docs/instrumentation/php/automatic.md:65]: @chalin same as above

Comment (chalin) [reply content/en/docs/instrumentation/php/automatic.md:65]: Right. Not all our files are consistent.

The convention we've being using moving forward is that if a command has no output, we use `lang="shell"` (or `sh`) as you have above, but then omit the prompt `$`.  Any objections to such a change?

Comment (pdelewski) [reply content/en/docs/instrumentation/php/automatic.md:65]: no, just observation

**Review by chalin — APPROVED** [review]

**Pull request merged**